include/uv-darwin.h

```diff
@@ -52,6 +57,7 @@
   void* cf_member[2];                                                         \
   int cf_error;                                                               \
   uv_mutex_t cf_mutex;                                                        \
+  enum uv_fs_event_type_flag type_flag;                                       \
```

Comment:
We do have some reserved fields for keeping backwards ABI compatibility: https://github.com/libuv/libuv/commit/06f9e1438ed20dd83618bca78759ea8635a79289

To be used just if the parachute doesn't open.